index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <h1 id="title">A Survey Form</h1>
+      <p id="description">This is a survey form for Tech School</p>
+      <form id="survey-form">
+        <label id="name-lable">Name</label>
+        <input type="text" id="name" placeholder="Enter your name">
+        <label id="email-lable">Email</label>
+        <input type="email" id="email" placeholder="test@gmail.com">
+        <label id="number-label">Roll No</label>
+        <input type="number" id="number" placeholder="Enter Roll No">
+        <label>How do you rate our courses</label>
+        <select class="dropdown">
+          <option>Excellent</option>
+          <option>Very Good</option>
+          <option>Good</option>
+        </select>
+        <label>How do you rate our teachers</label>
+        <select class="dropdown">
+          <option>Excellent</option>
+          <option>Very Good</option>
+          <option>Good</option>
+        </select>
+        <label>How do you rate our facilities</label>
+        <select class="dropdown">
+          <option>Excellent</option>
+          <option>Very Good</option>
+          <option>Good</option>
+        </select>
```

Comment:
Please don't leave the <option> tag values. you should assign each <option> tag value in order to send data. here is an example`<option value="Excellent">Excellent</option>`